dll/cpl/desk/lang/en-US.rc

```diff
@@ -27,6 +27,7 @@ BEGIN
     LTEXT "&Placement", IDC_STATIC, 188, 128, 55, 9
     COMBOBOX IDC_PLACEMENT_COMBO, 188, 139, 55, 54, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     PUSHBUTTON "Customize &Desktop...", IDC_DESKTOP_CUSTOM, 8, 194, 90, 15
+    CONTROL "<A>Get more wallpapers</A>", IDC_GETMOREWALLS, "SysLink", 0x2C | WS_TABSTOP, 174, 200, 67, 12
```

Comment:
0x2C ? What is that?

Reply:
Vista styles that I want but we don't support yet

Reply:
Is that constant documented somewhere? (or it's a composite value of several flags)

Reply:
Several flags, they are on msdn

Reply:
Please add them into `<commctrl.h>`.

Reply:
Under normal style they would be hidden behind a #if version check anyway, does not help me here.

Reply:
Ok.

Reply:
> Under normal style they would be hidden behind a #if version check anyway, does not help me here.

What do you mean? They are defined in commctrl.h (at least the official one) with #if versions?
If not, please add them in commctrl.h and use them here unconditionally. If the comctl32 sees unknown flags it should just ignore them.
At the very very least, a comment saying what 0x2C equals to as flags combination should be added.

Reply:
MS SDK has them behind `#if (NTDDI_VERSION >= NTDDI_VISTA)`

Reply:
OK then please add a comment for the 0x2C what kind of flags it is equals to, _**OR**_, define these flags in the resource.h of rapps.

Reply:
"The text is right-justified" (not RTL text) because I want to align it to the right edge without having to fit each translation text size to the control size. I'll leave it off for now, can be added back later as needed.

Reply:
Good point (for the right-aligning with the right edge).